kitsune/coolsearch/static/coolsearch/js/dispatcher/SearchDispatcher.js

```diff
@@ -0,0 +1 @@
+export default new Flux.Dispatcher();
```

Comment:
There should only be one dispatcher. You can use the one in `kitsune/sumo/static/sumo/js/dispatcher.js`.